src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -929,8 +929,12 @@ internal static Cci.IGenericParameterReference Translate(TypeParameterSymbol par
             DiagnosticBag diagnostics,
             bool needDeclaration = false)
         {
+            if (fieldSymbol.IsTupleField)
```

Comment:
I think this should be an assert. If we forget to lower field accesses in some context (we had such bug in object initializers), it is better if assert below is triggered right away, than a situation with a scenario working for small tuples and not working with big tuples.